src/harness/util.py

```diff
@@ -306,7 +304,7 @@ def computePropagationAntennaModel(request):
         ppa = request['ppa']
 
         ARCSEC = 2
-        ppa_points = ComputePPAProtectionPoints(ppa['geometry']['coordinates'], ARCSEC)
+        ppa_points = GridPolygon(ppa['geometry']['coordinates'], ARCSEC)
```

Comment:
Are you sure it is not: `GridPolygon(ppa['geometry'], ARCSEC)` as I said previously?

Reply:
This is how the GeoJson is defined

    "features": [
		{	
			"type": "Feature",
			"properties": {},
			"geometry": {
			  "type": "Polygon",
			  "coordinates": [
}

Reply:
This is incorrect, as by doing this you are doing the GeoJSON decoding but in a wrong way.

Note that the GridPolygon routines are designed to work with shapely or GeoJSON. 
As a side effect they also work with:
 + a list of coordinates [(lat1,lng1),(lat2,lng2),..]: in which case a polygon will be created.
 + a list of a list of coordinatates: in which case we will get a multipolygon, but we have lost the notion of holes, which is present in the geoGson

In GeoJson, the first sub array is the outer ring, and the others are holes. It is recommended to use CCW for exterior and CW for interior but this is not mandatory for back compatibility, and you cannot rely on that in practice.

So basically by passing the coordinates, you are using:
 1) undocumented input of the GridPolygon routine (passing a list of a list) instead of Shapely or GeoJSON geometry
 2) breaking the detection of holes.

Moreover your PPA json test are wrong, and do not follow the GeoJSON spec. You need one more bracket around the coordinates.


: the Utils routines do not take a list of coordinates. In fact it works because then the shape([[40, -60], [...]) that has been used 


If it works it is unintended.
It takes